src/components/LoginForm/LoginForm.css

```
@@ -27,6 +26,13 @@
   padding: 1rem 0 2.8rem 0;
 }
 
+.loginAuth-container > form > .unmatch-error {
+font-weight: 700;
+font-size: 1rem;
+color: var(--error);
+padding-bottom: 1rem;
+}
```

I believe you miss one indentation, comparing to the other css rules that you wrote on this file

```suggestion
.loginAuth-container > form > .unmatch-error {
    font-weight: 700;
    font-size: 1rem;
    color: var(--error);
    padding-bottom: 1rem;
}
```